OpenHistoricalMap/leaflet-ohm-timeslider-v2
# 624 ISO 8601 and BCE + 626 Localization improvements

This brings two sets of improvements per feedback from Minh 1ec5

https://github.com/OpenHistoricalMap/issues/issues/626

- locale sensitive range picker
- date outputs from Intl.DateTimeFormat

https://github.com/OpenHistoricalMap/issues/issues/624

- decimaldate update to new specification
- range picker, replace negative years with a CE/BCE picker
- Change Date modal, rework as four-input UI like range pickers

https://github.com/OpenHistoricalMap/issues/issues/630

- Date chooser modal, now has French translations for the buttons

**Testing**

- check out the code `git clone https://github.com/OpenHistoricalMap/leaflet-ohm-timeslider-v2.git`
- switch to the new ISO-8601 branch: `git checkout bce-iso8601-compliance`
- start a Python web server: `python -m SimpleHTTPServer 9646` or `python3 -m http.server 9646`
- point browser at http://localhost:9646/ and try it out
